capsules/extra/src/nmea_i2c_mt.rs

```diff
@@ -0,0 +1,241 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+//! A capsule for I2C NMEA. This is used to read NMEA sentences from a GPS/GNSS
+//! device connected via I2C.
```

Comment:
This comment really makes it seem like this should _actually_ be a PR for adding a GPS/GNSS driver, not an NMEA driver

Reply:
`nmea.rs` is a generic NMEA capsule. `nmea_i2c_mt.rs` is a chip-specific implementation that provides NMEA sentences from a GPS/GNSS device. So in my mind, the comment makes sense here. Maybe I'm not understanding you though?

Reply:
I also suspect it would work with any NMEA device over I2C, but I only have a GNSS device to test against.

Maybe a good excuse to buy a boat :)